sql_script/create_tables.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS user (
+    user_id VARCHAR(64) PRIMARY KEY,
+    user_type VARCHAR(32) NOT NULL,
+    name VARCHAR(64)
+) ENGINE=InnoDB CHARACTER SET=utf8mb4 COLLATE=utf8mb4_general_ci;
+
+CREATE TABLE IF NOT EXISTS accounting (
+    id INT UNSIGNED PRIMARY KEY,
+    user_id VARCHAR(64) UNIQUE KEY,
```

Comment:
add column userType?
or we can have a PK in user table then save pk value as unique userID here
userID cannot be a UNIQUE KEY because a user may have multiple account datas

Reply:
You're right. I was confused when I coding this.